src/Antiforgery/test/DefaultAntiforgeryTokenGeneratorTest.cs

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+#nullable disable
```

Comment:
This one's a massive pain. I'm leaving this as an exercise for the future.

Reply:
I didn't enable null in Newtonsoft.Json tests because of the extra work involved. A couple of APIs in Newtonsoft.Json were marked as not null that otherwise should have allowed null. Null in tests would have caught those. However I think these kind of issues should be caught in previews or otherwise corrected in servicing.

tldr, nullable in tests is nice, but I don't think it is a requirement when difficult